Source/Headspring.BulkWriter/Mapping.cs

```diff
@@ -27,15 +27,13 @@ public Mapping(string destinationTableName, IEnumerable<PropertyMapping> propert
         [SuppressMessage("Microsoft.Reliability", "CA2000:Dispose objects before losing scope", Justification = "Caller is responsible for disposing.")]
         public IBulkWriter<TResult> CreateBulkWriter(string connectionString)
         {
-            if (null == connectionString)
-            {
-                throw new ArgumentNullException("connectionString");
-            }
+            return CreateBulkWriter(new BulkWriterOptions(connectionString));
+        }
 
-            if (0 == connectionString.Length)
-            {
-                throw new ArgumentException(Resources.Mapping_CreateBulkWriter_InvalidConnectionString, "connectionString");
-            }
+        [SuppressMessage("Microsoft.Reliability", "CA2000:Dispose objects before losing scope", Justification = "Caller is responsible for disposing.")]
+        public IBulkWriter<TResult> CreateBulkWriter(BulkWriterOptions options)
```

Comment:
Given that this is a public method, you should ensure that this parameter value is not null.